chapters/ch17.md

```diff
@@ -1,32 +1,33 @@
 # 17. If you have a large dev set, split it into two subsets, only one of which you look at
 ->
-
+# 17. Với một tập phát triển lớn, chia tập đó thành hai tập nhỏ và chỉ nhìn vào một tập
 
 Suppose you have a large dev set of 5,000 examples in which you have a 20% error rate. Thus, your algorithm is misclassifying ~1,000 dev images. It takes a long time to manually examine 1,000 images, so we might decide not to use all of them in the error analysis.
 ->
-
+Giả sử bạn có một tập phát triển lớn gồm 5000 mẫu, thu về tỉ lệ lỗi là 20%. Ở đây, thuật toán của bạn đang phân loại nhầm khoảng 1000 mẫu ảnh của tập phát triển. Sẽ rất lâu để phân tích thủ công 1000 ảnh này, vì vậy, ta có thể không sử dụng tất cả 1000 ảnh đó trong phân tích lỗi. 
 
 In this case, I would explicitly split the dev set into two subsets, one of which you look at, and one of which you don’t. You will more rapidly overfit the portion that you are manually looking at. You can use the portion you are not manually looking at to tune parameters.
 ->
+Trong trường hợp này, tôi sẽ chia tập phát triển thành hai phần rõ ràng: một tập sẽ được nhìn vào, tập còn lại thì không. Tập phát triển được nhìn thủ công sẽ bị overfit nhanh hơn tập còn lại. Bạn có thể sử dụng tập còn lại để điều chỉnh tham số.
```

Comment:
- đề xuất "`Thuật toán sẽ overfit phần được tập trung` nhanh hơn. Bạn có thể sử dụng `phần` còn lại để điều chỉnh tham số",
> ~Tập phát triển được nhìn thủ công sẽ bị overfit~ nhanh hơn ~tập còn lại~.  Bạn có thể sử dụng ~tập~ còn lại để điều chỉnh tham số

Reply:
- từ explitcity dịch là "riêng biệt" thì có vẻ hay hơn "rõ ràng"

Reply:
Từ "subset" trong cả chương nên dịch là "tập con" vì nó có ý nghĩa toán học rõ ràng.

cụm đó có thể dịch là "chia tập phát triển thành hai tập con không giao nhau"